dll/win32/shell32/stubs.cpp

```diff
@@ -36,15 +36,33 @@ SHFindComputer(LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2)
     return FALSE;
 }
 
-/*
- * Unimplemented
+struct LIMIT_INPUT_COMBO
+{
+    HRESULT hr;
+    IShellFolder *psf;
+};
+
+static BOOL CALLBACK _FindTheEditBox(HWND hwndEdit, LPARAM lParam)
+{
+    LIMIT_INPUT_COMBO *pLimitInput = (LIMIT_INPUT_COMBO *)lParam;
+    pLimitInput->hr = SHLimitInputEdit(hwndEdit, pLimitInput->psf);
+    return FALSE;
+}
+
+/*************************************************************************
+ *  SHLimitInputCombo [SHELL32.748]
+ *
+ * Sets limits on valid characters for a combobox control.
+ * This function works like SHLimitInputEdit, but the target is a combobox
+ * instead of a textbox.
  */
 EXTERN_C BOOL
 WINAPI
-SHLimitInputCombo(HWND hWnd, LPVOID lpUnknown)
+SHLimitInputCombo(HWND hWnd, IShellFolder *psf)
 {
-    FIXME("SHLimitInputCombo() stub\n");
-    return FALSE;
+    LIMIT_INPUT_COMBO LimitInput = { E_FAIL, psf };
+    EnumChildWindows(hWnd, _FindTheEditBox, (LPARAM)&LimitInput);
```

Comment:
This function seems to suggest that there is only ever one child window, so why enumerate it and not just fetch the child window directly?
Something like `::GetWindow(hWnd, GW_CHILD);`

Reply:
Simplified by using `GetTopWindow`.